frontend/src/components/7.Result/ResultPage.css

```diff
@@ -39,19 +41,13 @@
   background-image: -ms-linear-gradient(top, #F63D52, #9D1ED0);
   background-image: -o-linear-gradient(top, #F63D52, #9D1ED0);
   background-image: -webkit-gradient(to bottom, #F63D52, #9D1ED0);
-  -webkit-border-radius: 20px;
-  -moz-border-radius: 20px;
-  border-radius: 20px;
+
   color: #FFFFFF;
-  font-family: Brush Script MT;
+  font-family: monospace;
```

Comment:
We don't need the CSS for .ButtonReset anymore so you can delete all of the CSS related to it on this file.

Reply:
It has been removed.